tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/AWS/LambdaCommon.cs

```diff
@@ -243,5 +257,68 @@ private static void WriteRequestHeaders(WebRequest request, IDictionary<string,
             var proxyInstance = obj.DuckAs<ILambdaContext>();
             return proxyInstance?.ClientContext?.Custom;
         }
+
+        private class ConverterContractResolver : DefaultContractResolver
```

Comment:
Do you mind adding a short comment explaining why we use a custom resolver and a link to https://www.newtonsoft.com/json/help/html/Performance.htm ? Thanks!

Reply:
Adding the comment!